spec/03-plugins/35-azure-functions/01-access_spec.lua

```diff
@@ -175,5 +205,18 @@ for _, strategy in helpers.each_strategy() do
       assert(tonumber(res.headers["Content-Length"]) > 100)
     end)
 
+    it("service upstream uri and request uri can not influence azure function", function()
+      local res = assert(proxy_client:send {
+        method  = "GET",
+        path    = "/",
+        query   = { hello = "world" },
+        headers = {
+          ["Host"] = "azure3.com"
+        }
+      })
+
+      assert(tonumber(res.headers["Content-Length"]) > 100)
```

Comment:
Just a question, what does `res.headers["Content-Length") > 100` mean?

Reply:
That's no other means, assert that an approved response from Mockbin is expected.